fenix/app/src/main/java/org/mozilla/fenix/settings/SecretSettingsFragment.kt

```diff
@@ -67,6 +67,12 @@ class SecretSettingsFragment : PreferenceFragmentCompat() {
         requirePreference<Preference>(R.string.pref_key_custom_sponsored_stories_parameters).apply {
             isVisible = Config.channel.isNightlyOrDebug
         }
+
+        requirePreference<SwitchPreference>(R.string.pref_key_enable_tabs_tray_to_compose).apply {
```

Comment:
super nit: if this were placed with the other switches it would be more visually consistent